python_ta/transforms/ExprWrapper.py

```diff
@@ -25,23 +25,20 @@ class ExprWrapper:
     """
 
     node: astroid.NodeNG
-    types: Dict[str, str]
+    types: Dict[str, Union[str, z3.ExprRef]]
 
     def __init__(self, node: astroid.NodeNG, types=None):
         if types is None:
             types = {}
         self.types = types
+        self.node = node
 
-        if isinstance(node, astroid.Expr):
-            self.node = node.value  # take node attribute to be the value of the expression
-        elif isinstance(node, astroid.Assign):
-            self.node = node.value  # take node attribute as expression (right side) of assignment
-        elif isinstance(node, astroid.Arguments):
-            self.node = node  # take node attribute to be the function declaration node itself
-        else:
-            raise ValueError(
-                "'node' param must be an astroid expression, assignment, or arguments node."
-            )
+        if not isinstance(node, astroid.NodeNG):
```

Comment:
This check is not necessary, as the type contract requires the argument be an `astroid.NodeNG`

Reply:
Hi Professor, I am aware of the type contract. This check is used to pass the unit test `test_expr_wrapper_invalid_node` in `test_expr_wrapper`. Would you suggest me to delete or modify that test instead?

Reply:
@Raine-Yang-UofT I see, thanks for clarifying. Yes please remove that test.